### PR TITLE
Tdr 430 run git secrets

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: vars/tdr.groovy [hunk 1]
```diff
@@ -13,6 +13,16 @@ def runEndToEndTests(int delaySeconds, String stage, String buildUrl) {
     wait: false)
 }
 
+def runGitSecrets(repo) {
+  sh "set +e"
+  def exitCode = sh(script: "git-secrets --scan", returnStatus: true)
+  sh "set -e"
+  if(exitCode == 1) {
```
Comment (suzannehamilton) [inline vars/tdr.groovy]: Is it worth catching any non-zero exit code?

Comment (MancunianSam) [reply vars/tdr.groovy]: Yeah I think so. I've changed it.

File: vars/tdr.groovy [hunk 2]
```diff
@@ -13,6 +13,16 @@ def runEndToEndTests(int delaySeconds, String stage, String buildUrl) {
     wait: false)
 }
 
+def runGitSecrets(repo) {
+  sh "set +e"
+  def exitCode = sh(script: "git-secrets --scan", returnStatus: true)
```
Comment (suzannehamilton) [inline vars/tdr.groovy:18]: Just to check - does this print the output to the console so we can track down the secret?

**Review by suzannehamilton — APPROVED** [review]

File: vars/tdr.groovy (hunk 2 above)
Comment (MancunianSam) [reply vars/tdr.groovy:18]: Yeah, it does, you get something like this
```
test.txt:1:XXXXXXXXXXXXXXXXXXX
[ERROR] Matched one or more prohibited patterns

```

**Pull request merged**